packages/magento-product/components/ProductPageBreadcrumb/ProductPageBreadcrumb.tsx

```diff
@@ -0,0 +1,34 @@
+import { usePrevPageRouter } from '@graphcommerce/framer-next-pages'
+import { Breadcrumbs, Link, Typography } from '@mui/material'
+import { productPageCategory } from '../ProductPageCategory/productPageCategory'
+import { ProductPageBreadcrumbFragment } from './ProductPageBreadcrumb.gql'
+
+export function ProductPageBreadcrumb(props: ProductPageBreadcrumbFragment) {
+  const { categories, name } = props
+  const prev = usePrevPageRouter()
+
+  const category =
+    categories?.find((c) => `/${c?.url_path}` === prev?.asPath) ?? productPageCategory(props)
+
+  return (
+    <Breadcrumbs>
+      <Link underline='hover' color='inherit' href='/'>
+        Home
+      </Link>
```

Comment:
All @mui/material links should always be wrapped in a `import PageLink from 'next/link`
```suggestion
      <PageLink href='/' passHref>
        <Link underline='hover' color='inherit'>
          Home
        </Link>
      </PageLink>
```

`Home` should be translated with the <Trans /> component